spec/features/home_spec.rb

```diff
@@ -44,19 +44,19 @@
       end
     end
 
-    context '自分がチェックインしているとき' do
+    context '自分が記録しているとき' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.